platforms/software/build-init/build.gradle.kts

```diff
@@ -48,7 +48,6 @@ dependencies {
     }
     implementation(projects.pluginsJvmTestSuite)
     implementation(projects.serviceLookup)
-    implementation(projects.wrapperMain)
```

Comment:
🤔 Adding this dependency back fixes the `AllDistributionIntegrationSpec.allZipContents` test but breaks `projectHealth`